core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -68,6 +68,8 @@ oppia.controller('CreatorDashboard', [
       smText: 'Publish the exploration to receive statistics.'
     };
 
+    var CurrentView = constants.ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS.CARD;
```

Comment:
This should be currentViewType. Variables start with lowercase; follow existing codebase conventions please.

Reply:
$scope.myExplorationView changes to card view when the page is set to mobile view so I declared a variable which stores the previously selected type of view in CurrentView

Reply:
Ah I think you meant this as a reply to the other comment. In general please make sure to reply to all comments (just "Done" is fine in most cases) and ensure your reply is targeted to the specific comment in question.

Re your comment: thanks for the explanation. It seems to me like there are multiple concepts here that we need to separate really clearly:

- the current user setting
- what mode is actually being used to display things

and we need to name these clearly. Can we name the first one userDashboardDisplayPreference and the second one $scope.myExplorationsView (as it is now)? There should be no other variables relating to this concept. There should also be a comment somewhere relevant in the code to explain that we do not honour the display preference in mobile mode.

The cleanest way to handle this, btw, is probably to have a function called getMyExplorationsViewType() and have it work out the logic based on the display preference variable and the device orientation/type. That way you can just call getMyExplorationsViewType() any time you need to know what the view type is, and it can do all the calculations internally.